Resources/Prototypes/Entities/Mobs/NPCs/asteroid.yml

```diff
@@ -90,12 +90,12 @@
     - id: FoodMeatGoliath
       amount: 3
     - id: MaterialGoliathHide1
-  - type: Prying # DeltaV: let sentient goliaths pry doors
-    pryPowered: true
-    force: true
-    speedModifier: 1.5
-    useSound:
-      path: /Audio/Items/crowbar.ogg
```

Comment:
this is all still fucked

Reply:
Can you just shoot me what ti replace it with still rather new to this, and not even sure why I needed to put it back on

Reply:
It looks like there's a space before each line between lines 93-98 that's fucking it up

Reply:
Okay did that unfuck it

Reply:
Close but no cigar. Delete the dashes below line 93, and remove the space in front of line 93 
![Screenshot_20241002-000132](https://github.com/user-attachments/assets/561ac914-99f8-4a9a-a107-d20fb45e32d5)

Reply:
There we go